cockroachcloud/create-an-account.md

```diff
@@ -48,21 +54,16 @@ GitHub will verify your identity using [GitHub 2FA](https://docs.github.com/en/g
 ## Log in to your account
 
 <div class="filters clearfix">
-  <button class="filter-button page-level" data-scope="github">Log in with GitHub</button>
+  <button class="filter-button page-level" data-scope="github">Log in with Single Sign-On (SSO)</button>
   <button class="filter-button page-level" data-scope="email">Log in with email</button>
 </div>
 <p></p>
 
 <section class="filter-content" markdown="1" data-scope="github">
 
-If you have already [registered a new {{ site.data.products.db }} account](#register-a-new-account) using GitHub, you can log in to {{ site.data.products.db }}:
-
 1. Navigate to the [{{ site.data.products.db }} Log In page](https://cockroachlabs.cloud/clusters).
```

Comment:
If you take a look at this in the preview there is some formatting issues. I'm not 100% on what the issue is. Wondering if it is the liquid template in the link.